libs/agno/agno/vectordb/qdrant/qdrant.py

```diff
@@ -35,8 +40,35 @@ def __init__(
         host: Optional[str] = None,
         path: Optional[str] = None,
         reranker: Optional[Reranker] = None,
+        use_hybrid_search: bool = False,
```

Comment:
ok instead of this, as per the current design we have `search_type: SearchType = SearchType.vector` 
where:-
```py
class SearchType(str, Enum):
    vector = "vector"
    keyword = "keyword"
    hybrid = "hybrid"
```

we should stick to it here as well.